conda-recipes/fastBPE-feedstock/.ci_support/linux_python3.6.yaml

```diff
@@ -0,0 +1,16 @@
+c_compiler:
+- gcc
+c_compiler_version:
+- '8'
+channel_sources:
+- conda-forge,defaults
```

Comment:
probably don't need conda-forge on the sources here.

Reply:
sure, updating it now.